lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -441,7 +441,10 @@ defmodule Kernel.ParallelCompiler do
       IO.puts(["  ", String.pad_leading(file, max), " => " | inspect(mod)])
     end
 
-    IO.puts("")
+    IO.puts(
+      "\nEnsure there are no compile-time dependencies between those files " <>
+        "and that the modules they reference exist and are correctly named.\n"
```

Comment:
Minor nitpick: should we remove the trailing dot as we usually don't include them in error messages?

Reply:
I will delegate this to whatever @whatyouhide and @lexmag decide, haha!

Reply:
I was unsure because the general rule is to not include trailing dots, but for error messages with multiple sentences that include dots it looks a bit weird when the final sentence is not terminated. I will defer to professional nitpickers.

Reply:
I believe in such cases we don't include trailing period regardless if it is multiple sentences or not. :bowtie:

Reply:
I can't remember what we do but I agree with @ericmj, if there are multiple sentences and they end with trailing period, the last one should end with trailing period too.

Reply:
I think it is important to be consistent with what we already have ([1](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/src/elixir_aliases.erl#L179), [2](https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/dep.ex#L269), [3](https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/cli.ex#L49), and many more).
Also, the simpler the rule the better—"don't include trailing period" is smooth to follow.

Otherwise, let's identify and change all the other places. :bowtie:

Reply:
> Also, the simpler the rule the better—"don't include trailing period" is smooth to follow.

I agree. To me we should remove the trailing period from error messages.